backend/src/govsg/services/govsg.service.ts

```diff
@@ -170,20 +172,35 @@ export function uploadCompleteOnChunk({
       }
     })
 
-    await GovsgMessage.bulkCreate(records as Array<GovsgMessage>, {
-      transaction,
-      logging: (_message, benchmark) => {
-        if (benchmark) {
-          logger.info({
-            message: 'uploadCompleteOnChunk: ElapsedTime in ms',
-            benchmark,
-            campaignId,
-            action: 'uploadCompleteOnChunk',
-          })
-        }
-      },
-      benchmark: true,
-    })
+    const govsgMessages = await GovsgMessage.bulkCreate(
+      records as Array<GovsgMessage>,
+      {
+        transaction,
+        logging: (_message, benchmark) => {
+          if (benchmark) {
+            logger.info({
+              message: 'uploadCompleteOnChunk: ElapsedTime in ms',
+              benchmark,
+              campaignId,
+              action: 'uploadCompleteOnChunk',
+            })
+          }
+        },
+        benchmark: true,
+        returning: true,
+      }
+    )
+    await Promise.all(
+      govsgMessages.map((message) =>
+        GovsgVerification.create(
```

Comment:
Could I ask why we can't make use of `bulkCreate` here?

Reply:
Cus I was silly. Using `bulkCreate` now!